tools/release_notes.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
I would prefer to keep the name of `release-notes.py`, also it doesn't match the references above.

Reply:
I originally renamed it because pylint was complaining about the dash, but now it doesnt complain anymore, so no idea what that was about. Changed it back :)

Reply:
Thanks!